src/components/home-page-slide-form/home-page-slide-form.js

```diff
@@ -0,0 +1,346 @@
+              <TextField
+                data-cy={`${lang}Title`}
+                id={`${lang}Title`}
+                className={styles.textField}
+                variant='outlined'
+                label={config.labels.homePageSlide.title}
+                multiline
+                value={values[`${lang}Title`]}
+                onChange={handleChange}
+                error={touched[`${lang}Title`] && !!errors[`${lang}Title`]}
+              />
+              {touched[`${lang}Title`] && errors[`${lang}Title`] && (
+                <div
+                  data-cy={`${lang}Title-error`}
+                  className={styles.inputError}
+                >
+                  {errors[`${lang}Name`]}
+                </div>
+              )}
+              <TextField
+                data-cy={`${lang}Description`}
+                id={`${lang}Description`}
+                className={styles.textField}
+                variant='outlined'
+                label={config.labels.homePageSlide.description}
+                multiline
+                value={values[`${lang}Description`]}
+                onChange={handleChange}
+                error={
+                  touched[`${lang}Description`] &&
+                  !!errors[`${lang}Description`]
+                }
+              />
+              {touched[`${lang}Description`] && errors[`${lang}Description`] && (
+                <div
+                  data-cy={`${lang}Description-error`}
+                  className={styles.inputError}
+                >
+                  {errors[`${lang}Description`]}
+                </div>
+              )}
+            </Paper>
+          </TabPanel>
+        ))}
+
+        <Button
+          id='contactsBack'
+          component={Link}
+          to={config.routes.pathToHomePageSlides}
+          variant='outlined'
+          color='primary'
+          className={styles.returnButton}
+          data-cy='goBackButton'
+        >
+          {config.buttonTitles.GO_BACK_TITLE}
+        </Button>
+        <SaveButton
+          className={styles.saveButton}
+          data-cy='save'
+          type='submit'
+          title={config.buttonTitles.CREATE_SLIDE_TITLE}
+        />
+      </form>
+      <Typography variant='h1' className={styles.slideTitle}>
+        {preview}
+      </Typography>
+      <Paper
+        elevation={5}
+        className={styles.slideWrapper}
+      >
+        <Avatar variant='square'
+          className={styles.square}
+          src={
+            slideImage ||`${config.IMG_URL}${slide.images.large}`
+          }
+          color='primary'
+        >
+          <ImageIcon className={styles.slideIcon} />
+        </Avatar>
+        <div className={styles.slideContent}>
+          <div>
+            <h3>{values.ukTitle}</h3>
+            <p>{values.ukDescription}</p>
+          </div>
+          <p className={styles.discoverMore}> {discoverMoreTitle}
+            <span>&#8594;</span></p>
```

Comment:
Try to replace this hardcode unicode with a materialUI icon or add it to the config file

Reply:
Fixed